specification/deviceupdate/resource-manager/readme.go.md

```diff
@@ -4,7 +4,7 @@ These settings apply only when `--go` is specified on the command line.
 
 ```yaml $(go)
 go:
-  license-header: MICROSOFT_MIT_NO_VERSION
+  license-header: MICROSOFT_APACHE_NO_VERSION
```

Comment:
Should it be `MIT` to be consistent with RPSaaSMaster branch?

Reply:
Consistent with RPSaaSMaster, please see https://github.com/Azure/azure-rest-api-specs-pr/blob/RPSaaSMaster/specification/deviceupdate/resource-manager/readme.go.md
Done per the comment by @ArcturusZhang, see https://github.com/Azure/azure-rest-api-specs-pr/pull/3103#pullrequestreview-606908523